src/index.ts

```diff
@@ -244,6 +244,33 @@ export default class FocusOverlay {
     }
   }
 
+  /**
+   * The 'data-focus-offset' property let's you customize the default offset value of the
+   * focus box. The class name 'focus-overlay-offset' indicates whether a custom offset is
+   * currently set or not.
+   *
+   * @param {HTMLElement}
+   */
+  private setFocusBoxOffset(focusedElement: HTMLElement): void {
+    const { focusBox } = this;
+    if (focusBox) {
+      // If a custom offset is defined, remove the class and reset the CSS offset variable
+      // to the default value.
+      if (focusBox.classList.contains('focus-overlay-offset')) {
```

Comment:
shouldn't this if be inside of `if (focusOffset !== null)` (at the very beginning)? Or is it somehow possible that the element does not have the data attribute, but the class?

Reply:
Yes that is possible:

element 1 receives focus and has the attribute  data-focus-offset="0" --> the global focusbox gets a new property for the CSS variable. To indicate that the variable is different, the element also receives the class 'focus-overlay-offset'.

element 2 receives focus and does not have a data-focus-offset --> but the global focusbox element still has the 'focus-overlay-offset' (indicating that the offset is currently not the default value) --> therefore it will be reset to the default one.